daprdocs/content/en/operations/observability/metrics/metrics-overview.md

```diff
@@ -96,12 +105,7 @@ spec:
 
 When this configuration is applied, a recorded metric with the `method` label of `orders/a746dhsk293972nz` will be replaced with `orders/`.
 
-### Watch the demo
```

Comment:
Video was removed as users should leverage the new configuration option rather than create a (very complicated) set of regular expressions

Reply:
Does this new option completely remove the need for regular expressions?

Reply:
@ItalyPaleAle - Any reply to this question?